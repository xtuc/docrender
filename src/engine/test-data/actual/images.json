{
  "title": "Publish-Bot test - images",
  "body": {
    "content": [
      {
        "endIndex": 1,
        "sectionBreak": {
          "sectionStyle": {
            "columnSeparatorStyle": "NONE",
            "contentDirection": "LEFT_TO_RIGHT",
            "sectionType": "CONTINUOUS"
          }
        }
      },
      {
        "startIndex": 1,
        "endIndex": 9,
        "paragraph": {
          "elements": [
            {
              "startIndex": 1,
              "endIndex": 9,
              "textRun": {
                "content": "Middle:\n",
                "textStyle": {}
              }
            }
          ],
          "paragraphStyle": {
            "namedStyleType": "NORMAL_TEXT",
            "alignment": "START",
            "direction": "LEFT_TO_RIGHT"
          }
        }
      },
      {
        "startIndex": 9,
        "endIndex": 11,
        "paragraph": {
          "elements": [
            {
              "startIndex": 9,
              "endIndex": 10,
              "inlineObjectElement": {
                "inlineObjectId": "kix.8gp0xzhjpoz",
                "textStyle": {}
              }
            },
            {
              "startIndex": 10,
              "endIndex": 11,
              "textRun": {
                "content": "\n",
                "textStyle": {}
              }
            }
          ],
          "paragraphStyle": {
            "namedStyleType": "NORMAL_TEXT",
            "alignment": "CENTER",
            "direction": "LEFT_TO_RIGHT"
          }
        }
      },
      {
        "startIndex": 11,
        "endIndex": 18,
        "paragraph": {
          "elements": [
            {
              "startIndex": 11,
              "endIndex": 18,
              "textRun": {
                "content": "Right:\n",
                "textStyle": {}
              }
            }
          ],
          "paragraphStyle": {
            "namedStyleType": "NORMAL_TEXT",
            "direction": "LEFT_TO_RIGHT"
          }
        }
      },
      {
        "startIndex": 18,
        "endIndex": 20,
        "paragraph": {
          "elements": [
            {
              "startIndex": 18,
              "endIndex": 19,
              "inlineObjectElement": {
                "inlineObjectId": "kix.4maarkdiktxp",
                "textStyle": {}
              }
            },
            {
              "startIndex": 19,
              "endIndex": 20,
              "textRun": {
                "content": "\n",
                "textStyle": {}
              }
            }
          ],
          "paragraphStyle": {
            "namedStyleType": "NORMAL_TEXT",
            "alignment": "END",
            "direction": "LEFT_TO_RIGHT"
          }
        }
      },
      {
        "startIndex": 20,
        "endIndex": 21,
        "paragraph": {
          "elements": [
            {
              "startIndex": 20,
              "endIndex": 21,
              "textRun": {
                "content": "\n",
                "textStyle": {}
              }
            }
          ],
          "paragraphStyle": {
            "namedStyleType": "NORMAL_TEXT",
            "direction": "LEFT_TO_RIGHT"
          }
        }
      },
      {
        "startIndex": 21,
        "endIndex": 27,
        "paragraph": {
          "elements": [
            {
              "startIndex": 21,
              "endIndex": 27,
              "textRun": {
                "content": "Left:\n",
                "textStyle": {}
              }
            }
          ],
          "paragraphStyle": {
            "namedStyleType": "NORMAL_TEXT",
            "direction": "LEFT_TO_RIGHT"
          }
        }
      },
      {
        "startIndex": 27,
        "endIndex": 29,
        "paragraph": {
          "elements": [
            {
              "startIndex": 27,
              "endIndex": 28,
              "inlineObjectElement": {
                "inlineObjectId": "kix.w0x6rkemfm3",
                "textStyle": {}
              }
            },
            {
              "startIndex": 28,
              "endIndex": 29,
              "textRun": {
                "content": "\n",
                "textStyle": {}
              }
            }
          ],
          "paragraphStyle": {
            "namedStyleType": "NORMAL_TEXT",
            "direction": "LEFT_TO_RIGHT"
          }
        }
      },
      {
        "startIndex": 29,
        "endIndex": 30,
        "paragraph": {
          "elements": [
            {
              "startIndex": 29,
              "endIndex": 30,
              "textRun": {
                "content": "\n",
                "textStyle": {}
              }
            }
          ],
          "paragraphStyle": {
            "namedStyleType": "NORMAL_TEXT",
            "direction": "LEFT_TO_RIGHT"
          }
        }
      },
      {
        "startIndex": 30,
        "endIndex": 37,
        "paragraph": {
          "elements": [
            {
              "startIndex": 30,
              "endIndex": 37,
              "textRun": {
                "content": "Small:\n",
                "textStyle": {}
              }
            }
          ],
          "paragraphStyle": {
            "namedStyleType": "NORMAL_TEXT",
            "direction": "LEFT_TO_RIGHT"
          }
        }
      },
      {
        "startIndex": 37,
        "endIndex": 39,
        "paragraph": {
          "elements": [
            {
              "startIndex": 37,
              "endIndex": 38,
              "inlineObjectElement": {
                "inlineObjectId": "kix.yj1yi5lnzxjt",
                "textStyle": {}
              }
            },
            {
              "startIndex": 38,
              "endIndex": 39,
              "textRun": {
                "content": "\n",
                "textStyle": {}
              }
            }
          ],
          "paragraphStyle": {
            "namedStyleType": "NORMAL_TEXT",
            "direction": "LEFT_TO_RIGHT"
          }
        }
      },
      {
        "startIndex": 39,
        "endIndex": 40,
        "paragraph": {
          "elements": [
            {
              "startIndex": 39,
              "endIndex": 40,
              "textRun": {
                "content": "\n",
                "textStyle": {}
              }
            }
          ],
          "paragraphStyle": {
            "namedStyleType": "NORMAL_TEXT",
            "direction": "LEFT_TO_RIGHT"
          }
        }
      },
      {
        "startIndex": 40,
        "endIndex": 49,
        "paragraph": {
          "elements": [
            {
              "startIndex": 40,
              "endIndex": 49,
              "textRun": {
                "content": "cropped:\n",
                "textStyle": {}
              }
            }
          ],
          "paragraphStyle": {
            "namedStyleType": "NORMAL_TEXT",
            "direction": "LEFT_TO_RIGHT"
          }
        }
      },
      {
        "startIndex": 49,
        "endIndex": 51,
        "paragraph": {
          "elements": [
            {
              "startIndex": 49,
              "endIndex": 50,
              "inlineObjectElement": {
                "inlineObjectId": "kix.tv6vq2tuonr2",
                "textStyle": {}
              }
            },
            {
              "startIndex": 50,
              "endIndex": 51,
              "textRun": {
                "content": "\n",
                "textStyle": {}
              }
            }
          ],
          "paragraphStyle": {
            "namedStyleType": "NORMAL_TEXT",
            "direction": "LEFT_TO_RIGHT"
          }
        }
      }
    ]
  },
  "documentStyle": {
    "background": {
      "color": {}
    },
    "pageNumberStart": 1,
    "marginTop": {
      "magnitude": 72,
      "unit": "PT"
    },
    "marginBottom": {
      "magnitude": 72,
      "unit": "PT"
    },
    "marginRight": {
      "magnitude": 72,
      "unit": "PT"
    },
    "marginLeft": {
      "magnitude": 72,
      "unit": "PT"
    },
    "pageSize": {
      "height": {
        "magnitude": 841.8897637795277,
        "unit": "PT"
      },
      "width": {
        "magnitude": 595.27559055118115,
        "unit": "PT"
      }
    },
    "marginHeader": {
      "magnitude": 36,
      "unit": "PT"
    },
    "marginFooter": {
      "magnitude": 36,
      "unit": "PT"
    },
    "useCustomHeaderFooterMargins": true
  },
  "namedStyles": {
    "styles": [
      {
        "namedStyleType": "NORMAL_TEXT",
        "textStyle": {
          "bold": false,
          "italic": false,
          "underline": false,
          "strikethrough": false,
          "smallCaps": false,
          "backgroundColor": {},
          "foregroundColor": {
            "color": {
              "rgbColor": {}
            }
          },
          "fontSize": {
            "magnitude": 11,
            "unit": "PT"
          },
          "weightedFontFamily": {
            "fontFamily": "Arial",
            "weight": 400
          },
          "baselineOffset": "NONE"
        },
        "paragraphStyle": {
          "namedStyleType": "NORMAL_TEXT",
          "alignment": "START",
          "lineSpacing": 115,
          "direction": "LEFT_TO_RIGHT",
          "spacingMode": "COLLAPSE_LISTS",
          "spaceAbove": {
            "unit": "PT"
          },
          "spaceBelow": {
            "unit": "PT"
          },
          "borderBetween": {
            "color": {},
            "width": {
              "unit": "PT"
            },
            "padding": {
              "unit": "PT"
            },
            "dashStyle": "SOLID"
          },
          "borderTop": {
            "color": {},
            "width": {
              "unit": "PT"
            },
            "padding": {
              "unit": "PT"
            },
            "dashStyle": "SOLID"
          },
          "borderBottom": {
            "color": {},
            "width": {
              "unit": "PT"
            },
            "padding": {
              "unit": "PT"
            },
            "dashStyle": "SOLID"
          },
          "borderLeft": {
            "color": {},
            "width": {
              "unit": "PT"
            },
            "padding": {
              "unit": "PT"
            },
            "dashStyle": "SOLID"
          },
          "borderRight": {
            "color": {},
            "width": {
              "unit": "PT"
            },
            "padding": {
              "unit": "PT"
            },
            "dashStyle": "SOLID"
          },
          "indentFirstLine": {
            "unit": "PT"
          },
          "indentStart": {
            "unit": "PT"
          },
          "indentEnd": {
            "unit": "PT"
          },
          "keepLinesTogether": false,
          "keepWithNext": false,
          "avoidWidowAndOrphan": true,
          "shading": {
            "backgroundColor": {}
          },
          "pageBreakBefore": false
        }
      },
      {
        "namedStyleType": "HEADING_1",
        "textStyle": {
          "fontSize": {
            "magnitude": 20,
            "unit": "PT"
          }
        },
        "paragraphStyle": {
          "namedStyleType": "NORMAL_TEXT",
          "direction": "LEFT_TO_RIGHT",
          "spaceAbove": {
            "magnitude": 20,
            "unit": "PT"
          },
          "spaceBelow": {
            "magnitude": 6,
            "unit": "PT"
          },
          "keepLinesTogether": true,
          "keepWithNext": true,
          "pageBreakBefore": false
        }
      },
      {
        "namedStyleType": "HEADING_2",
        "textStyle": {
          "bold": false,
          "fontSize": {
            "magnitude": 16,
            "unit": "PT"
          }
        },
        "paragraphStyle": {
          "namedStyleType": "NORMAL_TEXT",
          "direction": "LEFT_TO_RIGHT",
          "spaceAbove": {
            "magnitude": 18,
            "unit": "PT"
          },
          "spaceBelow": {
            "magnitude": 6,
            "unit": "PT"
          },
          "keepLinesTogether": true,
          "keepWithNext": true,
          "pageBreakBefore": false
        }
      },
      {
        "namedStyleType": "HEADING_3",
        "textStyle": {
          "bold": false,
          "foregroundColor": {
            "color": {
              "rgbColor": {
                "red": 0.2627451,
                "green": 0.2627451,
                "blue": 0.2627451
              }
            }
          },
          "fontSize": {
            "magnitude": 14,
            "unit": "PT"
          }
        },
        "paragraphStyle": {
          "namedStyleType": "NORMAL_TEXT",
          "direction": "LEFT_TO_RIGHT",
          "spaceAbove": {
            "magnitude": 16,
            "unit": "PT"
          },
          "spaceBelow": {
            "magnitude": 4,
            "unit": "PT"
          },
          "keepLinesTogether": true,
          "keepWithNext": true,
          "pageBreakBefore": false
        }
      },
      {
        "namedStyleType": "HEADING_4",
        "textStyle": {
          "foregroundColor": {
            "color": {
              "rgbColor": {
                "red": 0.4,
                "green": 0.4,
                "blue": 0.4
              }
            }
          },
          "fontSize": {
            "magnitude": 12,
            "unit": "PT"
          }
        },
        "paragraphStyle": {
          "namedStyleType": "NORMAL_TEXT",
          "direction": "LEFT_TO_RIGHT",
          "spaceAbove": {
            "magnitude": 14,
            "unit": "PT"
          },
          "spaceBelow": {
            "magnitude": 4,
            "unit": "PT"
          },
          "keepLinesTogether": true,
          "keepWithNext": true,
          "pageBreakBefore": false
        }
      },
      {
        "namedStyleType": "HEADING_5",
        "textStyle": {
          "foregroundColor": {
            "color": {
              "rgbColor": {
                "red": 0.4,
                "green": 0.4,
                "blue": 0.4
              }
            }
          },
          "fontSize": {
            "magnitude": 11,
            "unit": "PT"
          }
        },
        "paragraphStyle": {
          "namedStyleType": "NORMAL_TEXT",
          "direction": "LEFT_TO_RIGHT",
          "spaceAbove": {
            "magnitude": 12,
            "unit": "PT"
          },
          "spaceBelow": {
            "magnitude": 4,
            "unit": "PT"
          },
          "keepLinesTogether": true,
          "keepWithNext": true,
          "pageBreakBefore": false
        }
      },
      {
        "namedStyleType": "HEADING_6",
        "textStyle": {
          "italic": true,
          "foregroundColor": {
            "color": {
              "rgbColor": {
                "red": 0.4,
                "green": 0.4,
                "blue": 0.4
              }
            }
          },
          "fontSize": {
            "magnitude": 11,
            "unit": "PT"
          }
        },
        "paragraphStyle": {
          "namedStyleType": "NORMAL_TEXT",
          "direction": "LEFT_TO_RIGHT",
          "spaceAbove": {
            "magnitude": 12,
            "unit": "PT"
          },
          "spaceBelow": {
            "magnitude": 4,
            "unit": "PT"
          },
          "keepLinesTogether": true,
          "keepWithNext": true,
          "pageBreakBefore": false
        }
      },
      {
        "namedStyleType": "TITLE",
        "textStyle": {
          "fontSize": {
            "magnitude": 26,
            "unit": "PT"
          }
        },
        "paragraphStyle": {
          "namedStyleType": "NORMAL_TEXT",
          "direction": "LEFT_TO_RIGHT",
          "spaceAbove": {
            "unit": "PT"
          },
          "spaceBelow": {
            "magnitude": 3,
            "unit": "PT"
          },
          "keepLinesTogether": true,
          "keepWithNext": true,
          "pageBreakBefore": false
        }
      },
      {
        "namedStyleType": "SUBTITLE",
        "textStyle": {
          "italic": false,
          "foregroundColor": {
            "color": {
              "rgbColor": {
                "red": 0.4,
                "green": 0.4,
                "blue": 0.4
              }
            }
          },
          "fontSize": {
            "magnitude": 15,
            "unit": "PT"
          },
          "weightedFontFamily": {
            "fontFamily": "Arial",
            "weight": 400
          }
        },
        "paragraphStyle": {
          "namedStyleType": "NORMAL_TEXT",
          "direction": "LEFT_TO_RIGHT",
          "spaceAbove": {
            "unit": "PT"
          },
          "spaceBelow": {
            "magnitude": 16,
            "unit": "PT"
          },
          "keepLinesTogether": true,
          "keepWithNext": true,
          "pageBreakBefore": false
        }
      }
    ]
  },
  "revisionId": "ALBJ4Lsnrs1spox6ZVVEIgKFFz8hVLFHXlmUbNcpNjH809ryl-_cfrLHVVklSZqGGQyNZOhkWcaYB3V_bvy21jA",
  "suggestionsViewMode": "SUGGESTIONS_INLINE",
  "inlineObjects": {
    "kix.4maarkdiktxp": {
      "objectId": "kix.4maarkdiktxp",
      "inlineObjectProperties": {
        "embeddedObject": {
          "imageProperties": {
            "contentUri": "https://lh7-rt.googleusercontent.com/docsz/AD_4nXdQWBfwce4u73xbSlFK1Cxno-5O8_QMCUXuyGdOBHUV6d33o45oCBW-5sUaq0T6gZpFF98l4v4snJGLM95_RnXmIqxr4i8qh5bQZI52pWc2-hoNy-ZYmvCGxLDQ7I0tFnFv-uRPZ4l0AK95dune4cakn_-M?key=L81KPS631quh92g5K29tLA",
            "cropProperties": {}
          },
          "embeddedObjectBorder": {
            "color": {
              "color": {
                "rgbColor": {}
              }
            },
            "width": {
              "unit": "PT"
            },
            "dashStyle": "SOLID",
            "propertyState": "NOT_RENDERED"
          },
          "size": {
            "height": {
              "magnitude": 83.15033222591363,
              "unit": "PT"
            },
            "width": {
              "magnitude": 82.875,
              "unit": "PT"
            }
          },
          "marginTop": {
            "magnitude": 9,
            "unit": "PT"
          },
          "marginBottom": {
            "magnitude": 9,
            "unit": "PT"
          },
          "marginRight": {
            "magnitude": 9,
            "unit": "PT"
          },
          "marginLeft": {
            "magnitude": 9,
            "unit": "PT"
          }
        }
      }
    },
    "kix.8gp0xzhjpoz": {
      "objectId": "kix.8gp0xzhjpoz",
      "inlineObjectProperties": {
        "embeddedObject": {
          "imageProperties": {
            "contentUri": "https://lh7-rt.googleusercontent.com/docsz/AD_4nXdQWBfwce4u73xbSlFK1Cxno-5O8_QMCUXuyGdOBHUV6d33o45oCBW-5sUaq0T6gZpFF98l4v4snJGLM95_RnXmIqxr4i8qh5bQZI52pWc2-hoNy-ZYmvCGxLDQ7I0tFnFv-uRPZ4l0AK95dune4cakn_-M?key=L81KPS631quh92g5K29tLA",
            "cropProperties": {}
          },
          "embeddedObjectBorder": {
            "color": {
              "color": {
                "rgbColor": {}
              }
            },
            "width": {
              "unit": "PT"
            },
            "dashStyle": "SOLID",
            "propertyState": "NOT_RENDERED"
          },
          "size": {
            "height": {
              "magnitude": 83.15033222591363,
              "unit": "PT"
            },
            "width": {
              "magnitude": 82.875,
              "unit": "PT"
            }
          },
          "marginTop": {
            "magnitude": 9,
            "unit": "PT"
          },
          "marginBottom": {
            "magnitude": 9,
            "unit": "PT"
          },
          "marginRight": {
            "magnitude": 9,
            "unit": "PT"
          },
          "marginLeft": {
            "magnitude": 9,
            "unit": "PT"
          }
        }
      }
    },
    "kix.tv6vq2tuonr2": {
      "objectId": "kix.tv6vq2tuonr2",
      "inlineObjectProperties": {
        "embeddedObject": {
          "imageProperties": {
            "contentUri": "https://lh7-rt.googleusercontent.com/docsz/AD_4nXdQWBfwce4u73xbSlFK1Cxno-5O8_QMCUXuyGdOBHUV6d33o45oCBW-5sUaq0T6gZpFF98l4v4snJGLM95_RnXmIqxr4i8qh5bQZI52pWc2-hoNy-ZYmvCGxLDQ7I0tFnFv-uRPZ4l0AK95dune4cakn_-M?key=L81KPS631quh92g5K29tLA",
            "cropProperties": {
              "offsetLeft": 0.5585586,
              "offsetRight": -5.551115e-17,
              "offsetBottom": 0.45163524
            }
          },
          "embeddedObjectBorder": {
            "color": {
              "color": {
                "rgbColor": {}
              }
            },
            "width": {
              "unit": "PT"
            },
            "dashStyle": "SOLID",
            "propertyState": "NOT_RENDERED"
          },
          "size": {
            "height": {
              "magnitude": 45.651367187499943,
              "unit": "PT"
            },
            "width": {
              "magnitude": 36.75,
              "unit": "PT"
            }
          },
          "marginTop": {
            "magnitude": 9,
            "unit": "PT"
          },
          "marginBottom": {
            "magnitude": 9,
            "unit": "PT"
          },
          "marginRight": {
            "magnitude": 9,
            "unit": "PT"
          },
          "marginLeft": {
            "magnitude": 9,
            "unit": "PT"
          }
        }
      }
    },
    "kix.w0x6rkemfm3": {
      "objectId": "kix.w0x6rkemfm3",
      "inlineObjectProperties": {
        "embeddedObject": {
          "imageProperties": {
            "contentUri": "https://lh7-rt.googleusercontent.com/docsz/AD_4nXdQWBfwce4u73xbSlFK1Cxno-5O8_QMCUXuyGdOBHUV6d33o45oCBW-5sUaq0T6gZpFF98l4v4snJGLM95_RnXmIqxr4i8qh5bQZI52pWc2-hoNy-ZYmvCGxLDQ7I0tFnFv-uRPZ4l0AK95dune4cakn_-M?key=L81KPS631quh92g5K29tLA",
            "cropProperties": {}
          },
          "embeddedObjectBorder": {
            "color": {
              "color": {
                "rgbColor": {}
              }
            },
            "width": {
              "unit": "PT"
            },
            "dashStyle": "SOLID",
            "propertyState": "NOT_RENDERED"
          },
          "size": {
            "height": {
              "magnitude": 83.15033222591363,
              "unit": "PT"
            },
            "width": {
              "magnitude": 82.875,
              "unit": "PT"
            }
          },
          "marginTop": {
            "magnitude": 9,
            "unit": "PT"
          },
          "marginBottom": {
            "magnitude": 9,
            "unit": "PT"
          },
          "marginRight": {
            "magnitude": 9,
            "unit": "PT"
          },
          "marginLeft": {
            "magnitude": 9,
            "unit": "PT"
          }
        }
      }
    },
    "kix.yj1yi5lnzxjt": {
      "objectId": "kix.yj1yi5lnzxjt",
      "inlineObjectProperties": {
        "embeddedObject": {
          "imageProperties": {
            "contentUri": "https://lh7-rt.googleusercontent.com/docsz/AD_4nXdQWBfwce4u73xbSlFK1Cxno-5O8_QMCUXuyGdOBHUV6d33o45oCBW-5sUaq0T6gZpFF98l4v4snJGLM95_RnXmIqxr4i8qh5bQZI52pWc2-hoNy-ZYmvCGxLDQ7I0tFnFv-uRPZ4l0AK95dune4cakn_-M?key=L81KPS631quh92g5K29tLA",
            "cropProperties": {}
          },
          "embeddedObjectBorder": {
            "color": {
              "color": {
                "rgbColor": {}
              }
            },
            "width": {
              "unit": "PT"
            },
            "dashStyle": "SOLID",
            "propertyState": "NOT_RENDERED"
          },
          "size": {
            "height": {
              "magnitude": 24.301025390625,
              "unit": "PT"
            },
            "width": {
              "magnitude": 24.301025390625,
              "unit": "PT"
            }
          },
          "marginTop": {
            "magnitude": 9,
            "unit": "PT"
          },
          "marginBottom": {
            "magnitude": 9,
            "unit": "PT"
          },
          "marginRight": {
            "magnitude": 9,
            "unit": "PT"
          },
          "marginLeft": {
            "magnitude": 9,
            "unit": "PT"
          }
        }
      }
    }
  },
  "documentId": "1MgiCHaeIacVgjo4S1bd308A1zyY5Q5s1fFQZQqgr1j8"
}

