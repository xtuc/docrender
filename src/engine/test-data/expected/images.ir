(
    content: [
        Clear,
        Paragraph([
            InlineText([
                Text("Middle:"),
            ], true),
        ]),
        Clear,
        Paragraph([
            Image("https://lh7-rt.googleusercontent.com/docsz/AD_4nXdQWBfwce4u73xbSlFK1Cxno-5O8_QMCUXuyGdOBHUV6d33o45oCBW-5sUaq0T6gZpFF98l4v4snJGLM95_RnXmIqxr4i8qh5bQZI52pWc2-hoNy-ZYmvCGxLDQ7I0tFnFv-uRPZ4l0AK95dune4cakn_-M?key=L81KPS631quh92g5K29tLA", ("82", "83"), [
                ("text-align", "center"),
                ("margin-left", "auto"),
                ("margin-right", "auto"),
            ]),
        ]),
        Clear,
        Paragraph([
            InlineText([
                Text("Right:"),
            ], true),
        ]),
        Clear,
        Paragraph([
            Image("https://lh7-rt.googleusercontent.com/docsz/AD_4nXdQWBfwce4u73xbSlFK1Cxno-5O8_QMCUXuyGdOBHUV6d33o45oCBW-5sUaq0T6gZpFF98l4v4snJGLM95_RnXmIqxr4i8qh5bQZI52pWc2-hoNy-ZYmvCGxLDQ7I0tFnFv-uRPZ4l0AK95dune4cakn_-M?key=L81KPS631quh92g5K29tLA", ("82", "83"), [
                ("text-align", "right"),
            ]),
        ]),
        Clear,
        Paragraph([]),
        Paragraph([
            InlineText([
                Text("Left:"),
            ], true),
        ]),
        Clear,
        Paragraph([
            Image("https://lh7-rt.googleusercontent.com/docsz/AD_4nXdQWBfwce4u73xbSlFK1Cxno-5O8_QMCUXuyGdOBHUV6d33o45oCBW-5sUaq0T6gZpFF98l4v4snJGLM95_RnXmIqxr4i8qh5bQZI52pWc2-hoNy-ZYmvCGxLDQ7I0tFnFv-uRPZ4l0AK95dune4cakn_-M?key=L81KPS631quh92g5K29tLA", ("82", "83"), []),
        ]),
        Clear,
        Paragraph([]),
        Paragraph([
            InlineText([
                Text("Small:"),
            ], true),
        ]),
        Clear,
        Paragraph([
            Image("https://lh7-rt.googleusercontent.com/docsz/AD_4nXdQWBfwce4u73xbSlFK1Cxno-5O8_QMCUXuyGdOBHUV6d33o45oCBW-5sUaq0T6gZpFF98l4v4snJGLM95_RnXmIqxr4i8qh5bQZI52pWc2-hoNy-ZYmvCGxLDQ7I0tFnFv-uRPZ4l0AK95dune4cakn_-M?key=L81KPS631quh92g5K29tLA", ("24", "24"), []),
        ]),
        Clear,
        Paragraph([]),
        Paragraph([
            InlineText([
                Text("cropped:"),
            ], true),
        ]),
        Clear,
        Paragraph([
            Image("https://lh7-rt.googleusercontent.com/docsz/AD_4nXdQWBfwce4u73xbSlFK1Cxno-5O8_QMCUXuyGdOBHUV6d33o45oCBW-5sUaq0T6gZpFF98l4v4snJGLM95_RnXmIqxr4i8qh5bQZI52pWc2-hoNy-ZYmvCGxLDQ7I0tFnFv-uRPZ4l0AK95dune4cakn_-M?key=L81KPS631quh92g5K29tLA", ("36", "45"), []),
        ]),
        Clear,
        Paragraph([]),
    ],
    styles: {
        "pbNORMAL_TEXT": [
            ("line-height", "1.7249999"),
        ],
    },
)