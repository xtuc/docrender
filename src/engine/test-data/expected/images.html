<style>
  .pbNORMAL_TEXT {
    line-height: 1.7249999;
  }
</style>
<p>Middle:</p>
<p>
  <img
    style="text-align: center; margin-left: auto; margin-right: auto"
    src="https://test.com/i/c0b01de66a16cbbf4e1d43883e4f66fd26bd1015b6bebf37364742f7a9de7440"
    height="83"
    width="82"
  />
</p>
<p>Right:</p>
<p>
  <img
    style="text-align: right"
    src="https://test.com/i/c0b01de66a16cbbf4e1d43883e4f66fd26bd1015b6bebf37364742f7a9de7440"
    height="83"
    width="82"
  />
</p>
<p>Left:</p>
<p>
  <img
    src="https://test.com/i/c0b01de66a16cbbf4e1d43883e4f66fd26bd1015b6bebf37364742f7a9de7440"
    height="83"
    width="82"
  />
</p>
<p>Small:</p>
<p>
  <img
    src="https://test.com/i/c0b01de66a16cbbf4e1d43883e4f66fd26bd1015b6bebf37364742f7a9de7440"
    height="24"
    width="24"
  />
</p>
<p>cropped:</p>
<p>
  <img
    src="https://test.com/i/c0b01de66a16cbbf4e1d43883e4f66fd26bd1015b6bebf37364742f7a9de7440"
    height="45"
    width="36"
  />
</p>
